wxMSW: wxWakeUpIdle() does nothing if there's no top-level window.

In that case, call wxWakeUpMainThread() as a fall-back, otherwise events
sent from a sub-thread to the main thread may never get processed.

Signed-off-by: Anders Larsen <al@alarsen.net>
---

 src/msw/app.cpp |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)


diff --git a/src/msw/app.cpp b/src/msw/app.cpp
index a4f6edf..1a9b812 100644
--- a/src/msw/app.cpp
+++ b/src/msw/app.cpp
@@ -820,6 +820,10 @@ void wxApp::WakeUpIdle()
             }
         }
     }
+#if wxUSE_THREADS
+    else
+        wxWakeUpMainThread();
+#endif // wxUSE_THREADS
 }
 
 // ----------------------------------------------------------------------------
